workers/common/src/main/kotlin/common/OrtRunService.kt

```diff
@@ -118,6 +119,10 @@ class OrtRunService(
             config = config,
             scanners = scannerRun.scanners
         )
+
+        getScannerJob(scannerRun.scannerJobId)?.let { job ->
```

Comment:
I once more propose to use `also` instead of `let` as it better fits the intended semantics.